py-polars/src/series/construction.rs

```diff
@@ -233,17 +233,28 @@ impl PySeries {
     ) -> PyResult<Self> {
         let val = vec_extract_wrapped(val);
         let out = Series::new(name, &val);
-        match out.dtype() {
-            DataType::List(list_inner) => {
-                let out = out
-                    .cast(&DataType::Array(
-                        Box::new(inner.map(|dt| dt.0).unwrap_or(*list_inner.clone())),
-                        width,
-                    ))
-                    .map_err(PyPolarsErr::from)?;
-                Ok(out.into())
+        if out.len() == 0 {
+            // no inner list present
+            let out = out
```

Comment:
@ritchie46 I'm not sure if there's a more straightforward way of creating an empty Series with a specified Array dtype.

Reply:
Yes, you should use `Series::new_empty()` in that case. :)

Reply:
done.